api/swagger.yaml

```diff
@@ -2004,6 +2004,20 @@ definitions:
           compatibility.
         x-nullable: true
         $ref: "#/definitions/OCIDescriptor"
+      Manifests:
+        description: |
+            Manifests is a list of image manifests available in this image.  It
```

Comment:
I think there's a  double space here before `It`

Reply:
Yep, thanks